package.json

```diff
@@ -19,6 +19,7 @@
     "github-markdown-css": "^4.0.0",
     "global": "^4.4.0",
     "json-loader": "^0.5.7",
+    "less-loader": "4.1.0",
```

Comment:
please notify us to yarn install 😂

Reply:
okok, please yarn install before run dev

Reply:
👍